packages/fx-core/src/plugins/resource/aad/plugin.ts

```diff
@@ -587,4 +598,20 @@ export class AadAppForTeamsImpl {
 
     return requiredResourceAccessList;
   }
+
+  public async scaffold(ctx: PluginContext): Promise<AadResult> {
+    if (isAadManifestEnabled() && !isSPFxProject(ctx.projectSettings)) {
```

Comment:
I remember that only scaffold() of plugins that activated will be called. AAD plugin should not activate for spfx projects. Maybe we can remove the check? You may confirm with @jayzhang